Clinical-Genomics/gens
# individual_id linking from home sample list

This PR adds a functionality or fixes a bug.

**How to test**:
1. visit a case that loads nicely from scout-stage, go back to home page in gens and note that the link back to the case is broken on `master`
2. on this branch, rinse and repeat and note link working
<img width="1247" alt="Screenshot 2023-05-08 at 12 00 08" src="https://user-images.githubusercontent.com/758570/236796061-ca79eec1-9ee6-4013-8207-c3499b73dc63.png">

As for the actions, note deprecation warnings missing from Checks.

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
